ligo/src/registryDAO.mligo

```diff
@@ -120,29 +160,27 @@ let registry_DAO_decision_lambda (proposal, extras : proposal * contract_extra)
     proposal_metadata = proposal.metadata
     } in
   let proposal_key = to_proposal_key (propose_param, proposal.proposer) in
-  match Map.find_opt "agoraPostID" proposal.metadata with
-  | Some (packed_b) ->
+  match extract_proposal(proposal.metadata) with
+  | Normal_proposal diff ->
+      let ce_after_registry_update = apply_diff(diff, extras) in
+      let ce_after_registry_affected_update =
+        apply_diff_affected(proposal_key, diff, ce_after_registry_update) in
+        (([] : operation list), ce_after_registry_affected_update)
```

Comment:
In most cases we reassing a new value to the same name (to "update" it), I think we should do that here as well:
```suggestion
      let extras = apply_diff(diff, extras) in
      let extras = apply_diff_affected(proposal_key, diff, extras) in
      (([] : operation list), extras)
```

Reply:
Changed.